### ISSUE TITLE
Clarify my vision for the site

### ISSUE THREAD

**Issue by vegantriathlete** [vegantriathlete/how-to-brand-your-website#3]
What will the site do for me?

**Comment by vegantriathlete** [comment]
I think that I will NOT try to monetize this site directly. For any monetization, I will use the site to drive people to other places such as the Facebook group. I don't know if I'll end up doing any Udemy courses on veganism. So, I don't know if I will end up even caring about collecting email addresses.

My primary focus will be on ensuring that the site has negligible ongoing costs.
- I want to build it so that it can serve pages very quickly from cache so that it won't take a hit if it gets a lot of visitors.
- I will not allow visitors to sign in to the site or contribute anything to the site.

I would say the main thing that I want to get out of the site is to establish myself as an authority in this niche and to build a following through social media.

**Comment by vegantriathlete** [comment]
I think that something else I will get out of the site is a project that I can teach in one of my Drupal Udemy courses. So, I could be monetizing the site indirectly that way.

**Comment by kpalana** [comment]
Never say never. I think you could definitely use this project in other Udemy Drupal and web courses. I could also very well see you eventually doing a course on Veganism or raising vegan kids. Or maybe an ebook is on the horizon. These may all play off each other at some point.

**Comment by vegantriathlete** [comment]
Oooh, an ebook!
And you gave me the idea of using affiliate links to cookbooks.

Either of those options allow me to monetize the site indirectly as do your other suggestions. I'm not saying that the site won't help me to make money in some fashion. I'm saying that I'm not looking to have things blatantly on the site (like ads). I'm comfortable with affiliate links that are embedded within content, though.

I don't know if I want to provide some type of email opt-in. I could justify that type of thing on the grounds of notifying people when I've added a new recipe, new blog post, etc. Yes, I know it's possible to set up an RSS feed so that people could find that out. But, then they wouldn't have to give me their email address. I think it's probably worthwhile to come up with a way to collect email addresses. There's money in a list!

**Comment by kpalana** [comment]
Yes! I sent my FIRST newsletter out today to my little list I have been collecting. I know what I need to do to add more to my list (more tempting opt-ins and different ones for different pages) so just have to find the time for that.

You should definitely do an opt-in Maybe give away some mini PDF eBook (even 10 pages) that helps people with something related to veganism. It could be something like top recipes all vegan kids love or similar... or... Vegan treats for your child's birthday party that EVERY guest (assuming non-vegan) will love.
